src/web-worker/web-worker.js

```diff
@@ -60,5 +60,7 @@ self.addEventListener('message', function(event) {
 
   /** @type {FromWorkerMessageDef} */
   const message = {method, returnValue, id};
+  // `message` may only contain vlaues or objects handled by the
+  // structured clone algorithm.
```

Comment:
Add link to https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm here.